class ResourceNotAvailableError(Exception):
    pass

class TimeoutError(Exception):
    pass
